src/server/node_manager.c

```diff
@@ -900,12 +900,11 @@ post_discard_job(job *pjob, mominfo_t *pmom, int newstate)
 		sprintf(log_buffer, ndreque, downmom);
 		log_event(PBSEVENT_JOB, PBS_EVENTCLASS_JOB, LOG_INFO,
 			pjob->ji_qs.ji_jobid, log_buffer);
-		account_jobend(pjob, pjob->ji_acctrec, PBS_ACCT_RERUN);
+		force_reque(pjob);
 		if (pjob->ji_acctrec) {
 			free(pjob->ji_acctrec);	/* logged, so clear it */
 			pjob->ji_acctrec = NULL;
 		}
```

Comment:
If force_reque() is logging, maybe this should go into force_reque()?

Reply:
force_reque is calling account_jobend which does the logging.  I just reordered what was here to prevent logging twice.  Not all places that call account_jobend or force_reque free ji_acctrec, so I don't want to add it to within force_reque nor account_jobend.